### PR TITLE
Add the March 2020 newsletter

### PR DESCRIPTION
**Pull request by notlee**

### CODE REVIEW

File: _posts/2020-04-01-newsletter.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+	</strong>
+</abbr> {{page.description}}
+
+## Top Things
+
+These are some of the bigger things we've done over the last month.
+
+### Remote First
+
+It's hard to believe we were largely working from an office at the beginning of this month. A reminder that we're here to help. Our level of support is the same, and we’re happy to help via a hangout or Slack any time. 😊
```
Comment (chee) [inline _posts/2020-04-01-newsletter.md]: great :)

Comment (JakeChampion) [reply _posts/2020-04-01-newsletter.md]: _any time_, should we remove that? We are happy to help when we are working, whatever those hours are 😅

Comment (chee) [reply _posts/2020-04-01-newsletter.md]: that’s a good point.  this might be a good place to mention Office Hours too

Comment (notlee) [reply _posts/2020-04-01-newsletter.md:21]: Good point! 😅 I'll just remove that. Office hours are mentioned below 😊 

```suggestion
It's hard to believe we were largely working from an office at the beginning of this month. A reminder that we're here to help. Our level of support is the same, and we’re happy to help via a hangout or Slack. 😊
```

**Pull request merged**